_config.yml

```diff
@@ -23,3 +23,5 @@ github: # recreate gh-pages api
   repository_nwo: archivesspace/tech-docs
   owner_url: https://github.com/archivesspace
   owner_name: ArchivesSpace
+  edit_base_url: https://github.com/archivesspace/tech-docs/edit/master/
+  new_issue_base_url: https://github.com/archivesspace/tech-docs/issues/new?title=Problem+on+
```

Comment:
How about directing to the tech-docs project in our Jira? We would encourage people to create a Jira account which is a good first step towards taking part in the project and I think it is in general a good idea to keep our communication within the same tool.

I thought we could direct them to https://archivesspace.atlassian.net/jira/software/projects/TD/ where people can search for an already submitted issue that possibly describes the problem they are facing and then create an issue.

Reply:
@thimios didn't see a notification about this. Just pushed changes linking to https://archivesspace.atlassian.net/jira/software/projects/TD/issues since the top level board in your url above doesn't work.

Reply:
I think any link to the jira project where people can search the list of issues and create a new one is good. :+1: